### DISCUSSION TITLE
Resuming collection

### DISCUSSION THREAD

**Discussion by margauxw** [cjbarrie/academictwitteR#303] [Q&A]
### Please confirm the following

- [X] I have searched the existing issues
- [X] The behaviour of the program is deviated from what is described in the documentation.
- [X] I can reproduce this problem for more than one time.
- [x] This is NOT a 3-digit error -- it does not display an error message like `something went wrong. Status code: 400.`
- [ ] This is a 3-digit error and I have consulted the [Understanding API errors](https://cran.r-project.org/web/packages/academictwitteR/vignettes/academictwitteR-errors.html) vignette and the suggestions do not help.

### Describe the bug

If I do resume_collection it only pulls once because it has more than N = 100

### Expected Behavior

continue with the specified N in the original query

### Steps To Reproduce

resume_collection with N = inf in the original query

### Environment

.

**Comment by chainsawriot** [comment]
@margauxw This is not a bug and please refer to #217
